gui/src/3D/materials/mrdl/mrdlBackplateMaterial.ts

```diff
@@ -217,7 +217,7 @@ export class MRDLBackplateMaterial extends PushMaterial {
         this._activeEffect.setFloat("_Iridescence_Intensity_", 0.45);
         this._activeEffect.setFloat("_Iridescence_Edge_Intensity_", 1);
         this._activeEffect.setDirectColor4("_Iridescence_Tint_", new Color4(1, 1, 1, 1));
-        this._activeEffect.setTexture("_Iridescent_Map_", new Texture("https://github.com/rgerd/TestAssets/blob/master/PNG/BackplateIridescence.png", this.getScene()));
+        this._activeEffect.setTexture("_Iridescent_Map_", new Texture("https://raw.githubusercontent.com/rgerd/TestAssets/master/PNG/BackplateIridescence.png", this.getScene()));

```

Comment:
Could this be moved in the babylon assets repo ?

Reply:
also I wonder if it should not be embedded in the code in some ways or configurable through a static for instance ? @RaananW, any thoughts ?

Reply:
Yes, to assets and have it configurable for local hosting.

Reply:
The asset can be moved to repo, (Created this PR: https://github.com/BabylonJS/Assets/pull/51) and I can update this file once that is merged.